data/campaigns/Eastern_Invasion/scenarios/05_Northern_Outpost.cfg

```diff
@@ -1,453 +1,1124 @@
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "I think I may be able to help. Gweddry, I believe I can cast a spell that will reveal the bandits when our troops enter a village. However, this magic will require my full attention; I won’t be able to help you in battle."
-        [/message]
-
-        [message]
-            speaker=Gweddry
-            message= _ "That sounds good; it is time to drive out some bandits. Dacyn, cast your spell!"
-        [/message]
-
-        [message]
-            speaker=Owaec
-            message= _ "I believe there is a leader behind these outlaws; kill him and we will face no further resistance. I will place my clansmen at your service."
-        [/message]
-
-        [allow_recruit]
-            side=1
-            type=Horseman
-        [/allow_recruit]
-
-        [message]
-            speaker=narrator
-            image=wesnoth-icon.png
-            message= _ "You may now recruit horsemen!"
-        [/message]
-
-        [message]
-            speaker=Lanar-Skal
-            message= _ "Go, go far from here! But me and my kind will chase you, and kill you!"
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "It seems the undead have caught up with us! We will have to kill them too if we want the villagers to be safe. Well, I will begin casting the spell."
-        [/message]
-
-        {MOVE_UNIT id=Dacyn 13 14}
-
-        {DACYN_CASTING_SPELL_ANIMATION_OBJECT}
-
-        {MODIFY_UNIT id=Dacyn variables.casting_the_spell yes}
+        name=side 2 turn end  # do this at the end to delay Owaec's initial recruit, to make the achievement possible
+        # this also makes Owaec fight at night, which lets us give him more gold while still keeping his army weak
+        {MOVE_UNIT id=Raraery    18 20}
+        {MOVE_UNIT id=Raetharvan 18 21} # manually move peasant near Owaec's keep, so he doesn't run ahead and die early
+        {MOVE_UNIT id=Owaec      17 21} # manually move Owaec to the keep, since he's limited to only
+        {ZONE_GUARDIAN 2 Raraery 12 17 12 17 0} # sit on the forward river castle, and don't move
     [/event]
-
-    # Dacyn remains busy until the bandits are defeated
     [event]
-        name=side 1 turn refresh
+        name=side 2 turn
         first_time_only=no
-        [filter_condition]
-            [have_unit]
-                id=Dacyn
-                [filter_wml]
-                    [variables]
-                        casting_the_spell="yes"
-                    [/variables]
-                [/filter_wml]
-            [/have_unit]
-        [/filter_condition]
-
-        [modify_unit]
-            [filter]
-                id=Dacyn
-            [/filter]
+        {RESET_SIDE_AI 2 aggressive 0.8 0.25}
+
+        {MODIFY_SIDE_AI (2) ([avoid] # avoid everywhere except the road to the dark adept
+        x=1-14,15-17,18-99,   0-99
+        y= 0-5, 0-15, 0-99,  23-99
+    [/avoid])}
```

Comment:
CI check: indentation is broken from this line onwards.

Reply:
(this is an example of #1397, for reference)